src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int HASHCODE_FOR_NULL_VALUE = 0;
+    private static final int DEFAULT_SCALING_NUMBER = 2;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_SIZE = 0;
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        thresholdCounter();
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int bucket = hash(key);
+        Node<K, V> node = findKey(table[bucket], key, value);
+        if (node == null) {
+            table[bucket] = new Node<>(key,value);
+        } else if ((node.key == key) || (node.key != null && node.key.equals(key))) {
+            return;
```

Comment:
why are you exiting from the method? Instead of it, the value in the node should be replaced